blog/index.html

```diff
@@ -3,5 +3,11 @@
 index: false
 ---
 
+{% capture md %}
+You can also [Subscribe to Atom feeds](/blog/feeds/) in a feed reader.
+{% endcapture %}
+
+{{ md | markdownify }}
```

Comment:
This file has to be `.html`, not `.md`, for pagination to work properly. As I prefer keeping the pages Markdown when possible, I've used the `capture` & `markdownify` method here.